easybuild/easyblocks/w/wrf.py

```diff
@@ -158,7 +158,7 @@ def configure_step(self):
 
         # fetch selected build type (and make sure it makes sense)
         known_build_types = ['serial', 'smpar', 'dmpar', 'dm+sm']
-        self.parallel_build_types = ["dmpar", "smpar", "dm+sm"]
+        self.parallel_build_types = ["dmpar", "dm+sm"]
```

Comment:
@akesandgren This doesn't look OK to me... `smpar` is a parallel build type, isn't it?

Reply:
No, the parallel build type only applies to MPI builds, basically it's used to decide which tests to run and if mpirun is to be used. It's not used for building stuff. SO a more correct name would have been mpi_build_types.